peps/pep-0639.rst

```diff
@@ -668,12 +666,10 @@ A few additions will be made to the existing specifications.
 :term:`Installed project`\s
   The `Recording Installed Projects specification <installedspec_>`__ will be
   updated to reflect that if the ``Metadata-Version`` is ``2.4`` or greater
-  and one or more ``License-File`` fields is specified, the ``.dist-info``
+  and one or more ``License-File`` fields is specified, the installed ``.dist-info``
   directory MUST contain a ``licenses`` subdirectory which MUST contain
   the files listed in the ``License-File`` fields in the ``METADATA`` file
-  at their respective paths relative to the ``licenses`` directory,
-  and that any files in this directory MUST be copied from wheels
-  by install tools.
+  at their respective paths relative to the ``licenses`` directory.
```

Comment:
The clause "and that any files in this directory MUST be copied from wheels by install tools" has two potential interpretations:

- Files declared via the `License-File` Core Metadata field in the wheel distribution MUST be copied into this directory. This isn't _wrong_, but it is redundant with the earlier text. Simply stating that we're talking about the _installed_ `.dist-info` directory should suffice IMO.
- Only files declared in the `License-File` Core Metadata field are permitted to be in this directory. If this is the case, I'd suggest adding the word "only" somewhere.

I initially read the line under the second interpretation, but that seems wrong? I'd appreciate your thoughts @befeleme.

Reply:
Since I'm not the original author, let me provide another interpretation - how I see it: PEP doesn't specify nor forbid what _else_, except for the files listed under the `License-File` fields can be listed in the `.dist-info/licenses` directory, but it ensures that whatever is there (e.g. filled in by a build backend custom means), will always be installed with the copies of the software. Hence the partial redundancy of the wording.